projects/VRDR/DeathRecord_submissionProperties.cs

```diff
@@ -3685,8 +3685,8 @@ public string EducationLevelEditFlagHelper
         /// <para>// Getter:</para>
         /// <para>Console.WriteLine($"Birth Record identification: {ExampleDeathRecord.BirthRecordId}");</para>
         /// </example>
-        [Property("Birth Record Id", Property.Types.String, "Decedent Demographics", "Birth Record Identifier (i.e. Certificate Number).", true, IGURL.BirthRecordIdentifier, true, 16)]
-        [FHIRPath("Bundle.entry.resource.where($this is Observation).where(code.coding.code='BR')", "")]
+        [Property("Birth Record Id", Property.Types.String, "Decedent Demographics", "Birth Record Identifier (i.e. Certificate Number).", true, ProfileURL.BirthRecordIdentifier, true, 16)]
+        [FHIRPath("Bundle.entry.resource.where($this is Observation).where(code.coding.code='childbirthrecordidentifier')", "")]
```

Comment:
Related to the comment I had for the observations, is this correct? I thought childbirthrecordidentifier would go with the BirthRecordChildId, State, and Year fields

Reply:
It is definitely confusing.   But I think it is correct.
BirthRecordIdentifier; // decedent is infant child, link to birth certificate of decedent.  Code is childbirthrecordidentifier.

Decedent Birth Record Identifier.
new in STU3 -- decedent is mother, link is cert from recent delivery. Code is decedentbirthrecordidentifier

Reply:
Should Birth Record Identifier Child be renamed to Decedent Birth Record Identifier so the names are more consistent or is it not worth the effort to update the IG? [https://build.fhir.org/ig/HL7/vrdr/StructureDefinition-vrdr-birth-record-identifier-child.html](https://build.fhir.org/ig/HL7/vrdr/StructureDefinition-vrdr-birth-record-identifier-child.html) 
If it's not worth the update, I think comments about what each identifier is used for could suffice reduce confusion

Reply:
No.   It is confusing.
BirthRecordIdentifier (Infant) --> decedent is infant.   The profile name was created in STU2 before we knew about linkage from the mother to her child's birth record.   Didn't want to change the name of the profile.
ChildBirthRecordIdentifier --> decedent is mother.  Linkage is to (mother's) child's birth record.

Too late to change the IG.